security/passage/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        FiloSottile passage 1.7.4a1
+categories          security
```

Comment:
please add "github.tarball_from" and set it to "releases" (preferred, if upstream supports this) or to "archive". In either case you will need to update the "checksums".

Reply:
`releases` only has `1.7.4a0` not `1.7.4a1`, with a year's difference between the two tags. Set to `archive`.